clang/lib/Driver/Driver.cpp

```diff
@@ -6407,6 +6531,9 @@ class OffloadingActionBuilder final {
   /// Flag set to true if all valid builders allow file bundling/unbundling.
   bool CanUseBundler;
 
+  /// Flag set to true if we are performing early full device compilation
+  bool LinkDevice;
```

Comment:
Can we name this `LinkDeviceEarly `or `LinkDeviceCodeEarly `or something?

Reply:
This var isn't even used after the move to use `-fno-sycl-rdc`.  Will clean up.